src/argilla/server/alembic/versions/e402e9d9245e_create_responses_table.py

```diff
@@ -36,6 +36,7 @@ def upgrade() -> None:
         sa.Column("values", sa.JSON, nullable=False),
         sa.Column("record_id", sa.Uuid, sa.ForeignKey("records.id", ondelete="CASCADE"), nullable=False, index=True),
         sa.Column("user_id", sa.Uuid, sa.ForeignKey("users.id", ondelete="SET NULL"), index=True),
+        sa.Column("status", sa.String, nullable=False, index=True),
```

Comment:
> Also, I've realized that alembic revision --autogenerate command cannot be used to generate [migrations automatically](https://alembic.sqlalchemy.org/en/latest/autogenerate.html)

In your opinion, what are the benefits of utilizing automatic migrations?

> Another doubt that I have, is that I've seen that the attributes nullable, index, etc are being set in the migrations files, but not in the mapped_column function of sqlalchemy.

In the same vein as the previous question, what are your views on the advantages of including these attributes in models?

Reply:
I think that I can answer both questions at the same time 😁

So if we include these attributes in the models, then `alembic` will be available to automatically pick up them and it will generate a statement in the migration script with all the columns constraints, index, etc.

I think that it's really beneficial to use `alembic` automatic migrations, because `alembic` is able to automatically detect the updates you have done in your model class, or in other words, it is able to generate a `diff` between the schema generated with all the previous revisions and the one you're about to generate, and it will only create statements for the things that need to be updated.

So in my opinion, it is better to let `alembic` generate the migration file automatically with the `SQLAlchemy` model metadata than to let a human generate it manually and increase the chances of forgetting to include something in the migration script.